database/migrations/1732793568886_create_departments_table.ts

```diff
@@ -0,0 +1,28 @@
+import { BaseSchema } from '@adonisjs/lucid/schema'
+
+export default class extends BaseSchema {
+  protected tableName = 'departments'
+
+  async up() {
+    this.schema.createTable(this.tableName, (table) => {
+      table.increments('id').notNullable()
```

Comment:
you don't need to specify `notNullable` on primary key

Reply:
Changed.